src/main/java/org/testcontainers/containers/CephContainer.java

```diff
@@ -56,6 +61,10 @@ public CephContainer(final String dockerImageName) {
     public CephContainer(final DockerImageName dockerImageName) {
         super(dockerImageName);
         dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
+        withCreateContainerCmdModifier(createContainerCmd ->
```

Comment:
i cant really see from the code what you said was the solution in the issue 

"Solution:
Set the wait strategy in the constructor of your container. Instead of in the configure. This way setting a strategy after calling new CephContainer or Super() will override the waitingStrategy."

can you elaborate how the solution matches the code. Because you are not setting wait strategy in constructor now. You are setting cmd modifier with ulimit settings

Reply:
I thought solving this in the constructor would be possible. Unfortunately with the variable log check that's not possible (because in the constructor ` this.cephBucket` isn't defined yet)
I have solved this by check what the waiting strategy is set in configure. (i.e. when `start()` is called)
If Strategy is default we set the logical strategy which is by logging. 
Otherwise we leave it as it's set by the user.